frontend/src/methods/index.ts

```diff
@@ -247,6 +247,27 @@ export const downloadMachineJoinConfig = async (
 
 type DocsType = 'talos' | 'omni' | 'k8s'
 
+/**
+ * @deprecated Update links to use getDocsLink instead where possible.
+ */
+export function getLegacyDocsLink(path?: string, options?: { talosVersion?: string }) {
+  const parts = ['https://talos.dev']
```

Comment:
This function became mandatory here because now that I load doc links from the API, I have to support the old format.

I guess they should be updated in talos, and then image factory and frontend should be updated to support them.

Are there any other locations that might be using these doc links that will require updating? 🤔